tests/test_cases/issue_134/test_reals.py

```diff
@@ -26,3 +26,21 @@ def assign_double(dut):
     log.info("Read back value %g" % got)
     if got != val:
         raise TestFailure("Values didn't match!")
+
+@cocotb.test(expect_error=cocotb.SIM_NAME in ["Icarus Verilog"])
+def assign_double(dut):
+    """
+    Assign a random floating point value, read it back from the DUT and check
+    it matches what we assigned
+    """
```

Comment:
The test name and comment are wrong here

Reply:
Noticed this right after I pushed. You're fast.